packages/esm-ward-app/src/ward-view-header/admission-requests-bar.component.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Movement } from '@carbon/react/icons';
+import styles from './admission-requests.scss';
+import { launchWorkspace, showNotification } from '@openmrs/esm-framework';
+import { useInpatientRequest } from '../hooks/useInpatientRequest';
+import { useParams } from 'react-router-dom';
+import { SkeletonIcon } from '@carbon/react';
+import { useTranslation } from 'react-i18next';
+
+const AdmissionRequestsBar = () => {
+  const { locationUuid } = useParams();
+  const { inpatientRequests, isLoading, error } = useInpatientRequest(locationUuid);
+  const admissionRequests = inpatientRequests?.filter((request) => request.type == 'ADMISSION');
+  const { t } = useTranslation();
+
+  if (isLoading) {
+    return <SkeletonIcon className={styles.skeleton} />;
+  }
+
+  if (error) {
+    showNotification({
+      kind: 'error',
+      title: t('errorLoadingPatientAdmissionRequests', 'Error Loading Patient Admission Requests'),
+      description: error.message,
+    });
```

Comment:
It's better to show the Inline Notification with a "Retry" action on the UI rather than showing a notification and then not showing anything.
Thanks!